### DISCUSSION TITLE
How to pull order level reimbursement report in sp api

### DISCUSSION THREAD

**Discussion by Arul-Cointab** [amzn/selling-partner-api-models#2816] [General]
Which report type to use to pull order level reimbursement report.

Basically in seller central we use Reports -> Fulfillment by Amazon - > Payment -> Order Level Reimbursement to manually download the report.

in sp api which report will provide this information

**Comment by sungolivia** [comment]
closest FBA reimbursement report is this:
<img width="798" alt="Screenshot 2023-02-09 at 4 20 45 PM" src="https://user-images.githubusercontent.com/7606170/217968844-b7a6d998-b5bc-4ce5-a1ad-ba0fcf8a6cc0.png">
https://developer-docs.amazon.com/sp-api/docs/report-type-values#fulfillment-by-amazon-fba-reports

Otherwise probably not supported at the moment :(
